interface/ceed-preconditioning.c

```diff
@@ -1506,6 +1509,9 @@ int CeedOperatorLinearAssembleQFunction(CeedOperator op, CeedVector *assembled,
   if (op->LinearAssembleQFunction) {
     // Backend version
     CeedCall(op->LinearAssembleQFunction(op, assembled, rstr, request));
+  } else if (op->op_fallback_parent && op->op_fallback_parent->LinearAssembleQFunction) {
+    // Backend version for fallback parent
+    CeedCall(CeedOperatorLinearAssembleQFunction(op->op_fallback_parent, assembled, rstr, request));
```

Comment:
I don't understand this tbh. From `CeedOperatorCreateFallback`, wouldn't `op->op_fallback_parent` be the same as `op`, thus `op->op_fallback_parent->LinearAssembleQFunction == op->LinearAssembleQFunction`?

Reply:
Different `op`. Here, `op_fallback_parent` would fallback to `op` for diagonal or full assembly, then diagonal or full assembly calls the function to assemble the QFunction, and here we're checking if the `op_fallback_parent` has a faster way to assembly the QFunction. (which should really come before checking if the `op` can assemble

Reply:
CodeCov showed me that I made the change in the wrong place. Hopefully this latest change helps clarify

Reply:
Ah, I was misreading before anyways. Now this makes sense.